spec/models/measure_collection_spec.rb

```diff
@@ -0,0 +1,36 @@
+RSpec.describe MeasureCollection do
+  subject(:collection) { described_class.new(measures, declarable) }
+
+  let(:declarable) { build(:commodity) }
+
+  describe '#filter' do
+    before do
+      allow(TradeTariffBackend).to receive(:service).and_return(service)
+    end
+
+    let(:service) { 'uk' }
```

Comment:
do we need this?

Reply:
See line 8 and the one spec that isn't interested in the service